front/pages/w/[wId]/data-sources/vaults/[vaultId]/categories/[category]/index.tsx

```diff
@@ -77,6 +121,39 @@ export default function Vault({
   systemVault,
 }: InferGetServerSidePropsType<typeof getServerSideProps>) {
   const router = useRouter();
+  const sendNotification = useContext(SendNotificationsContext);
+
+  const handleSelect = async (sId: string) => {
+    switch (category) {
+      case "apps": {
```

Comment:
Either we cannot land on this case or you should call `handleSelect` on line 86 (`onSelect` of `VaultAppsList`)!